src/targets/github.ts

```diff
@@ -127,10 +127,17 @@ export class GitHubTarget extends BaseTarget {
       };
     }
 
-    const { data: latestRelease } = await this.github.repos.getLatestRelease({
-      owner: this.githubConfig.owner,
-      repo: this.githubConfig.repo,
-    });
+    let latestRelease: { tag_name: string } | undefined = undefined;
+    try {
+      latestRelease = (
+        await this.github.repos.getLatestRelease({
+          owner: this.githubConfig.owner,
+          repo: this.githubConfig.repo,
+        })
+      ).data;
+    } catch {
```

Comment:
The blanket `catch` here may mask other potential issues such as authentication or temporary server unavailability. I feel like we should filter these.

Reply:
what would you filter here? To me it sounds reasonable to fall back to assume it is the latest release if something goes wrong here 🤔 But if you want to exclude something specific, we can of course do that, open for concrete suggestions (not really easy to test/try this...)

Reply:
We should not assume lack of a release on a 401 or 500 response. I'd even say unless we get a 200 with an explicit null in it, we should not assume anything?

Reply:
So you mean we should fail the whole release if we do not get a 200 here?

Reply:
Yes as otherwise you create an incomplete release or worse, mark a hotfix release as latest accidentally.

If you think this behavior should be optional, we should revise the flow and make this an explicit user choice or make a breaking release.